draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -124,30 +124,38 @@ In the remainder of the document, it is assumed that each browser execution cont
 ICE Candidate Gathering {#gathering}
 ----------------------------
 
-For any host ICE candidate gathered by a browsing context as part of {{RFC8445}} section 5.1.1, obfuscation of the candidate is done as follows:
+For any host candidate gathered by an ICE agent as part of {{RFC8445}} section 5.1.1, obfuscation of the candidate is done as follows:
 
-1. Check whether the context ICE agent registered a name resolving to the ICE host candidate IP address.
+1. Check whether there is a usable registered mDNS hostname resolving to the ICE host candidate's IP address.
 
-2. If the ICE agent registered the name, replace the IP address of the ICE host candidate with the name with ".local" appended to it. Expose the candidate and abort these steps.
+2. If there is a registered a hostname, replace the IP address of the ICE host candidate with the hostname. Expose the candidate and abort these steps.
 
-3. Generate a random unique name, typically a version 4 UUID as defined in {{RFC4122}}.
+3. Generate a unique name. The unique name SHOULD consist of a version 4 UUID as defined in {{RFC4122}}. The name MUST NOT contain any ".".
```

Comment:
I think this should be a MUST - this avoids the issue noted below.

Reply:
Changed to MUST.